test/local/routes/sms.js

```diff
@@ -259,8 +267,8 @@ describe('/sms', () => {
       assert.equal(sms.send.callCount, 1)
     })
 
-    it('did not call log.flowEvent', () => {
-      assert.equal(log.flowEvent.callCount, 0)
+    it('called log.flowEvent once', () => {
+      assert.equal(log.flowEvent.callCount, 1)
```

Comment:
Ha ha, so to continue our conversation from https://github.com/mozilla/fxa-auth-server/pull/1766#discussion_r108526723 :smile:, in this case the region / event name has already been asserted for success and error paths by previous test cases. All I'm interested in asserting here is that the general behaviour appears sane, I don't see the need to exhaustively re-assert every invariant.